demos/CMakeLists.txt

```diff
@@ -1,33 +1,104 @@
-project (nbody)
+# Policy required to enable MSVC_RUNTIME_LIBRARY property 
+cmake_policy(SET CMP0091 NEW)
 
-cmake_minimum_required(VERSION 3.2.2)
```

Comment:
Where does it say it?

Reply:
It's in the CMake documentation. Ultimately, because the minimum version requirement sets what policies are available among other things, it is recommended to put it first.

Reply:
It is recommended to put it in the top level cmake file as far as I know.

Reply:
Possibly, but in this case this file will possibly end up needing 3.11 (if we add the FetchContent), while the rest of the CMake can continue to work with older versions - this way, if a user doesn't have CMake 3.11, they can still build the samples and tests.